ntoskrnl/io/pnpmgr/devaction.c

```diff
@@ -1363,14 +1365,19 @@ IopSetServiceEnumData(
         return Status;
     }
 
-    if (kvInfo2->Type != REG_SZ || kvInfo2->DataLength <= sizeof(WCHAR))
+    if ((kvInfo2->Type != REG_SZ) ||
+        (kvInfo2->DataLength <= sizeof(WCHAR)) ||
```

Comment:
less than equal, or (strictly) less than ?

Reply:
The original code has `<=`, so I'll keep it to not change the logic